src/components/pages/Cases/CaseTable.js

```diff
@@ -164,13 +164,76 @@ export default function CaseTable(props) {
     return month + '/' + day + '/' + year;
   }
 
-  function changeSorter(sorter) {
-    console.log(sorter);
+  // state to keep track of filters being applied to the table (Initial cases section)
+  const [initialFilters, setInitialFilters] = useState([]);
+  // keeping track of filters applied to the appelate section
+  const [appFilters, setAppFilters] = useState([]);
+
+  // state to keep track of the current table being displayed
+  const [currentKey, setCurrentKey] = useState(1);
+
+  useEffect(() => {
+    // TODO: don't know what to put into this useEffect - current use is to keep the filter state in sync.
+    console.log(initialFilters);
+  }, [initialFilters]);
+
+  useEffect(() => {
+    // TODO: don't know what to put into this useEffect - current use is to keep the filter state in sync.
+    console.log(appFilters);
```

Comment:
Nice work! is there a way to have both appFilters and initialFilters in one useEffect?

Reply:
This is something that I didn't think of! I'll have to revisit this.